zulipterminal/ui_tools/boxes.py

```diff
@@ -166,6 +166,8 @@ def generic_autocomplete(self, text: str, state: Optional[int]
                 ('@_', self.autocomplete_mentions),
                 ('@', self.autocomplete_mentions),
                 ('#', self.autocomplete_streams),
+                ('#*', self.autocomplete_streams),
```

Comment:
I can see the appeal for `#*` autocompleting to a list of all the available streams.

However, I am not sure if we should support `#*[a-z]` as a prefix. `#*s` autocompleting to `#**stream**` seems a bit fallacious given that the original prefix is `#**`.

Reply:
I agree that completion with the single-star after `#` and `@_` are maybe confusing after the different behavior for plain `@`. The original motivation was based on `#<text>` currently completing to eg `#**<text>**`, so text between the two cases are reasonable too?